src/sentry/integrations/jira/integration.py

```diff
@@ -377,21 +380,25 @@ def get_issue_type_meta(self, issue_type, meta):
 
     def get_create_issue_config(self, group, **kwargs):
         fields = super(JiraIntegration, self).get_create_issue_config(group, **kwargs)
-        params = kwargs.get('params', {})
-
-        # TODO(jess): update if we allow saving a default project key
 
+        params = kwargs.get('params', {})
+        default_project = self.get_project_defaults(group.project_id).get('project')
         client = self.get_client()
+
         try:
-            resp = client.get_create_meta(params.get('project'))
+            resp = client.get_create_meta(default_project)
         except ApiUnauthorized:
             raise IntegrationError(
                 'Jira returned: Unauthorized. '
                 'Please check your configuration settings.'
             )
 
         try:
+            # TODO(lb): I have no idea what meta is here; just going off tests
             meta = resp['projects'][0]
+            for project in resp['projects']:
+                if project['id'] == default_project:
+                    meta = project
```

Comment:
This represents my best guess for what meta is. @macqueen if you could tell me if this looks reasonable.